### PR TITLE
update setup_experiment and related scripts for jet

### PR DESCRIPTION
**Pull request by delippi**
This PR is just meant to update the setup_experiment and related scripts in rrfs-test. This script is a good place for developers to start setting up their test case.

### CODE REVIEW

**Review by ShunLiu-NOAA — DISMISSED** [review]

File: rrfs-test/scripts/templates/run_mpasjedi_template.sh [hunk 1]
```diff
@@ -18,6 +18,8 @@ if [[ $hostname == "h" ]]; then
   platform="hera"
 elif [[ $hostname == "o" ]]; then
   platform="orion"
+elif [[ $hostname == "f" ]]; then
+  platform="jet"
```
Comment (guoqing-noaa) [inline rrfs-test/scripts/templates/run_mpasjedi_template.sh]: FYI, at the computing nodes, the hostname does not necessarily return strings leading with "f", "h" etc.

**Review by guoqing-noaa — DISMISSED** [review]
LGTM. Thanks!

File: rrfs-test/scripts/templates/run_mpasjedi_template.sh (hunk 1 above)
Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: What do you suggest this should be?

Comment (guoqing-noaa) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: At this moment, we may use a similar method as what is used by global workflow and rrfs-workflow, i.e. to check specific folders. Here is an example:
https://github.com/NOAA-EMC/rrfs-workflow/blob/dev-sci/ush/fix_rrfs_locations.sh

Hercules and Orion share the same file system but I recalled that `hostname` at their computing nodes returns distinguishable strings between them.

Comment (guoqing-noaa) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: ok, we already have a powerful script in RDASApp:
`RDASApp/ush/detect_machine.sh`

Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: Yep, I just found that and was going to use it. Thank you!

Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: That makes a lot of these scripts a lot simpler. Thanks for the suggestion.

Comment (ShunLiu-NOAA) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: @delippi Have you completed the test on all three machines?

Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: Give me a moment and I will confirm on all three.

Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: @ShunLiu-NOAA, everything still works on Orion and Hera. I had a strange failure on Jet that I need to figure out.

Comment (delippi) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: @ShunLiu-NOAA, it seems to be working now on Jet as well. I ***think*** the issue was one of Jet's files systems (lsf4) was having issues and something in module load was trying to reach something staged there.

Comment (guoqing-noaa) [reply rrfs-test/scripts/templates/run_mpasjedi_template.sh]: @delippi  You are right. /lfs4 had performance issues in the past few days and Admin took Jet down to fix it. Fortunately, it was successfully fixed yesterday.

**Review by guoqing-noaa — APPROVED** [review]

**Comment by ShunLiu-NOAA** [comment]
@delippi Is there any other update needed for this PR?

**Comment by delippi** [comment]
Hi @ShunLiu-NOAA, it should be all set.

**Review by ShunLiu-NOAA — APPROVED** [review]

**Pull request merged**